lib/components/intent.js

```diff
@@ -470,6 +474,22 @@ Intent.prototype.createAlias = function(alias, roomId) {
     });
 };
 
+/**
+ * Set the presence of this user.
+ * @param {string} presence One of "online", "offline" or "unavailable".
+ * @param {string} status_msg The status message to attach.
+ * @return {Promise}
+ */
+Intent.prototype.setPresence = function(presence, status=undefined) {
```

Comment:
`status` vs `status_msg` in the docs

Reply:
http://matrix-org.github.io/matrix-js-sdk/0.10.3/module-client-MatrixClient.html#setPresence Not sure which I should be following

Reply:
`Math.random() <= 0.5 ? "status" : "status_msg"` (aka: pick one :p )